src/cache.js

```diff
@@ -33,7 +33,7 @@ export function getTmp () {
  * @return {WriteStream} - Write Stream
  */
 export function write () {
-	return fs.WriteStream(getTmp())
+	return fs.createWriteStream(getTmp())
```

Comment:
What's the idea behind this change (https://github.com/nodejs/node/blob/master/lib/fs.js#L1827).